bindings/netframework/ElectionGuard.NetFramework/Tests/ElectionGuard.Tests/Mocks/ManifestGenerator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using ElectionGuardInterop;
+
+namespace ElectionGuard.Tests.Mocks
+{
+    public class ManifestGenerator
+    {
+        public static Manifest GetMinimalFakeJeffersonCountyManifest()
+        {
+            var gpUnits = new List<GeopoliticalUnit>
+        {
+            new GeopoliticalUnit("jefferson-county", "Jefferson County", ReportingUnitType.county)
```

Comment:
for our examples we'll continue to use the jefferson county test data to help context switching between json and code